test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -896,6 +814,268 @@ public void sendNotification() {
         
         verify(notificationsService).sendNotificationToUser(STUDY.getStudyIdentifier(), HEALTH_CODE, message);
     }
+
+    // Creating an account and supplying an externalId
+    
+    @Test
+    public void createExternalIdValidatedRequiredWithValue() {
+        setupExternalIdTest(true, true);
+        
+        participantService.createParticipant(STUDY, CALLER_ROLES, PARTICIPANT, false);
+        
+        verifyIdReservation(EXTERNAL_ID);
+        verifyNotSetAsOption();
```

Comment:
A quick comment here explaining why when externalIdValidationEnabled=true means we don't set participant options. Otherwise, this leads to further confusion down the line.